### DISCUSSION TITLE
Inline-Editable Content Blocks

### DISCUSSION THREAD

**Discussion by theAdhocracy** [craftcms/ckeditor#203] [Ideas] [answered]
Currently, it seems that the only way to display an embedded/nested piece of content within a CK Editor field is as a card. Are there any plans to make that configurable, as with nested content in Matrix fields?

In particular, is it likely that we would ever be able to configure content to appear as inline-editable within a CK Editor field, or is that not technically feasible?

(Sorry if this has already been asked, but I wasn't able to find any prior conversations.)

**Comment by brandonkelly** [comment, marked as answer]
There’s no plans to add support for inline-editable blocks, as it would add a huge amount of complexity.
